Problems/1629/ans_1629.py

```diff
@@ -0,0 +1,17 @@
+#https://www.acmicpc.net/problem/1629
+# (A^B)%C 로 하게 되면 시간 복잡도가 올라가서 분할로 접근해야 함
+
+A, B, C = list(map(int, input().split()))
```

Comment:
```suggestion
A, B, C = map(int, input().split())
```
굳이 리스트로 안묶어도 됨 (갯수가 명백할경우)

Reply:
아 글쿤요